src/python/blazingmq/dev/reserveport.py

```diff
@@ -127,4 +127,4 @@ def reserve_port() -> typing.Iterator[TcpAddress]:
 
         address = tcp_address(sockname[0], sockname[1])
 
-    yield address
+        yield address
```

Comment:
👍🏻

Reply:
This line was correct before this PR. I already fixed this, since the port allocation logics were broken on fast machines like M Macs.

Reply:
https://github.com/bloomberg/blazingmq/pull/377